src/main/java/uk/gov/hmcts/divorce/common/event/SwitchedToSoleFinalOrder.java

```diff
@@ -64,7 +64,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
         configBuilder
             .event(SWITCH_TO_SOLE_FO)
-            .forStateTransition(EnumSet.of(AwaitingFinalOrder, AwaitingJointFinalOrder, FinalOrderRequested), FinalOrderRequested)
+            .forStateTransition(EnumSet.of(AwaitingFinalOrder, AwaitingJointFinalOrder), FinalOrderRequested)
```

Comment:
FinalOrderRequested was added to the enum set in [NFDIV-2409](https://tools.hmcts.net/jira/browse/NFDIV-2409)
are we sure we want to remove it now?